cli/args/flags.rs

```diff
@@ -4373,7 +4374,7 @@ impl CommandExt for Command {
     ).arg(
       Arg::new("unstable-detect-cjs")
         .long("unstable-detect-cjs")
-        .help("Reads the package.json type field in a project to treat .js files as .cjs")
+        .help("Treats ambiguous .js, .jsx, .ts, .tsx files as CommonJS modules in more cases")
         .value_parser(FalseyValueParser::new())
         .action(ArgAction::SetTrue)
         .hide(true)
```

Comment:
Unhide?

Reply:
I thought so too, but all of the unstable flags seem hidden

Reply:
Oh, just make sure it's shown when you do `deno run --help=unstable`